public_html/wp-content/mu-plugins/blocks/assets/src/blocks-store.js

```diff
@@ -0,0 +1,206 @@
+/**
+ * Implements wc-blocks-store.
+ */
+
+/**
+ * WordPress dependencies
+ */
+const { registerStore, dispatch } = wp.data;
+const { apiFetch } = wp;
+const { addQueryArgs } = wp.url;
```

Comment:
`[MUST]` - I know this is a WIP right now, but don't forget to run the linter later on to catch un-aligned assignments and other things.

Reply:
commited js:lint fixes. resolving this conversation now.